lib/game.js

```diff
@@ -16,7 +16,7 @@ Game = {
 
 	create_player: function(socket){
 		Logger.debug('Game.create_player');
-		Game.players.push(new Player(socket));
+		Game.players.push(new PlayerController(socket));
```

Comment:
This is just temporary. Eventually we'll still want to store a list of player, not the player controller object. That'd be weird.